src/V3EmitMk.cpp

```diff
@@ -31,11 +31,557 @@ class EmitMk final {
 public:
     // METHODS
 
+    struct FileOrConcatenatedFilesList final {
+        std::string fileName{};
+        std::vector<std::string> concatenatedFileNames{};
+
+        // Concatenating file score for debugTestConcatenation
+        std::int64_t dbgScore = 0;
+
+        bool isConcatenatingFile() const { return !concatenatedFileNames.empty(); }
+    };
+
+    struct FileNameWithScore final {
+        std::string name;
+        std::int64_t score;
```

Comment:
```suggestion
        std::string m_filename;
        std::int64_t m_score;
```

Likewise m_ all members.